src/app/cli/src/transition_frontier_persistence_rpc_worker.ml

```diff
@@ -0,0 +1,109 @@
+open Core
+open Async
+open Coda_base
```

Comment:
Transition_frontier_persistence already has a dependence on `coda_base`. I do think this functor is unnecessary and `Transition_frontier_persistence` should define its own RPC worker.